pages/common/git-archive.md

```diff
@@ -1,28 +1,28 @@
 # git archive
 
-> Create an archive of files from a named tree.
+> Create an archive of files from a tree.
 > More information: <https://git-scm.com/docs/git-archive>.
 
 - Create a tar archive from the contents of the current HEAD and print it to `stdout`:
 
 `git archive --verbose HEAD`
 
-- Create a zip archive from the current HEAD and print it to `stdout`:
+- Use the `zip` format and report progress [v]erbosely:
 
-`git archive --verbose --format zip HEAD`
+`git archive -{{v|-verbose}} --format zip HEAD`
 
-- Same as above, but write the zip archive to file:
+- [o]utput the zip archive to a specific file:
 
-`git archive --verbose --output {{path/to/file.zip}} HEAD`
+`git archive -v -{{o|-output}} {{path/to/file.zip}} HEAD`
 
-- Create a tar archive from the contents of the latest commit on a specific branch:
+- Create a `tar` archive from the contents of the latest commit of a specific branch:
 
-`git archive --output {{path/to/file.tar}} {{branch_name}}`
+`git archive -o {{path/to/file.tar}} {{branch_name}}`
```

Comment:
This progression to showing only the short option also seems fine to me in this case since it's clear from the previous example what `-o` means.